### ISSUE TITLE
kubeshark install has wrong documentation and concerning wording. 

### ISSUE THREAD

**Issue by domdorn** [kubeshark/kubeshark#1264]
As the install instructions on https://docs.kubeshark.co/en/install just say I have to use `kubeshark tap` to install (which I think is wrong), I took a try at `kubeshark install`

```
⇒  ./kubeshark install
Hello! This command can be used to install Kubeshark Pro edition on your Kubernetes cluster.
Please run:
	kubeshark install -o | kubectl apply -n kubeshark -f -

or use helm chart as described in https://getkubeshark.io/docs/installing-kubeshark/centralized-installation
```

there are two problems with this:
 - first its talking about a Kubeshark Pro edition where I as a new user have now idea what this is and what consequences it has. 
 - second, its mentioning a URL that is not valid anymore, as getkubeshark.io server is not reachable from my location (austria/europe)

as a potentially new user, I'm currenlty lost.. the screenshot on this repo looks nice, but I have no way of using this / trying this out with our k8s/openshift cluster/namespace.

**Comment by alongir** [comment]
@domdorn I agree there is confusion. Allow me to clarify:
1. To run `kubeshark` open source project, all you need is to run: `kubeshark tap -A` (I added the -A to indicate all namespaces, otherwise you'll tap only the default namespace). So `tap` is the correct command to run the `kubeshark` open source version.
2. As you can read [here](https://kubeshark.co/mizu-is-now-kubeshark), `kubeshark` (fka Mizu) was part of a larger product suite that also included a Pro version. The Pro version comes from a completely different codebase and docker images.  To install the Pro version, you need to run `kubeshark install` and follow the on-screen instructions. The Pro version includes more features than the open source version, it is not open source. We are now in the process of re-evaluating the architectures and the most popular features as well as refactoring a significant portion of the codebase, as part of our short term roadmap. You can use the Pro version free of charge. If you like certain features let us know and we will introduce them into `kubeshark`

**Comment by domdorn** [comment]
I'm sorry, I don't understand why this issue got assigned to me. 
I'm a "potential" first time (new) user.. at the moment, this product is not usable to me personally, nor in the context of my customer engagement (I'm a freelancer) where I only have rights to a certain k8s/openshift namespace (although I have a wide range of rights on that particular namespace).

I hope the bug report is showing you where some work is needed. However, I'm not able to support you in any other way than to try out a newer version in the future.

**Comment by alongir** [comment]
@domdorn it was by mistake. Sorry for that. The bug report was very helpful.

**Comment by tomaaron** [comment]
Thanks for explaining @alongir. I'm also happy to test out newer version as long as I could see the manifests or even better install a Helm Chart.

**Comment by Karabiy** [comment]
@alongir so, is kubeshark actually free? So far I wanted to test it on my cluster, but run into fact that it requires me to put a license - and only 2 nodes supported, which means basically 1 node when you're running on AWS. 
It's not clear if this tool is for everybody to use, or is it enterprise network analyzer which I can install on my cluster and upgrade to Pro version for additional features. 
It would be great in case if you could clarify if kubeshark is even usable for free, or it's only paid version? 
If it's possible to run it completely for free, in community mode, how is this achievable?

**Comment by Compy** [comment]
Agreed that the wording around these is incredibly confusing. To use it on a 3 node cluster requires a license, but the popup says "A complimentary PRO license is automatically provisioned when you sign up.", however I've signed up AND signed in, and I'm still getting the popup.

**Comment by alongir** [comment]
@Compy thanks for the feedback! You are correct! I confirm there's an issue with the automatic provisioning. It should be fixed today/tomorrow. I'll report here, once it does. Once this bug is fixed, and upon sign up, Kubeshark can be used across any size cluster with limited capacity.

**Comment by Compy** [comment]
@alongir Thanks Alon! I definitely look forward to trying it out.

**Comment by alongir** [comment]
UPDATE:
While we do see relics of a problem, we can't seem to reproduce it. We may have resolved the issue with our tests.
I suggest to try each of the following remedies by order and see if it helps:
1. Try to log in to the dashboard again and see if the issue was (magically) resolved.
2. Log in to https://console.kubeshark.co. This will trigger the automatic provisioning process. Try to log in to the dashboard.
3. Copy the license and use with: `--set license=<license>`
4. Reach out to support@kubeshark.co and will send you a temporary license.

LMK if that works or not.

**Comment by Ruakij** [comment]
I, too, installed kubeshark via helm and am just confused.
The popup on my own hosted system claims i cannot run it, disabling "cloudLicense" (whatever that may be) just results in the application going into "demo" mode, which would be fine, if i could actually login or setup a user for myself at all.

The value documentation for helm is incomplete and/or wrong because certain keys exist, but have different names. Some arent really understandable and seem to contradict other documentation sources (e.g. for auth)

Then there is this "Pro" license, i have no idea what that is, why i need it or if "complimentary" means it would be free for me or not as the linked pricing page has costs listed for the pro license based on API-request (that i make against my own server? wtf?)

I dont see why this external stuff is necessary at all.
